src/api/common_api.js

```diff
@@ -790,6 +790,8 @@ module.exports = {
             }
         },
 
+        // nsfs 
+
         fs_path: {
```

Comment:
Seems like we can get rid of this schema type and just use type string everywhere we need.

Reply:
removed.